api/envoy/extensions/tracers/opentelemetry/samplers/v3/dynatrace_sampler.proto

```diff
@@ -28,19 +27,21 @@ message DynatraceSamplerConfig {
   // The value can be obtained from the Envoy deployment page in Dynatrace.
   int32 cluster_id = 2;
 
-  // The HTTP URI to fetch the sampler configuration (root spans per minute). For example:
+  // The HTTP service to fetch the sampler configuration from the Dynatrace API (root spans per minute). For example:
   //
   // .. code-block:: yaml
   //
-  //    http_uri:
-  //      uri: <tenant>.dev.dynatracelabs.com/api/v2/samplingConfiguration
-  //      cluster: dynatrace
-  //      timeout: 10s
+  //    http_service:
+  //      http_uri:
+  //        cluster: dynatrace
+  //        uri: <tenant>.dev.dynatracelabs.com/api/v2/samplingConfiguration
+  //        timeout: 10s
+  //      request_headers_to_add:
+  //      - header:
+  //          key : "authorization"
+  //          value: "Api-Token dt..."
   //
-  config.core.v3.HttpUri http_uri = 3;
-
-  // The access token to fetch the sampling configuration from the Dynatrace API
-  string token = 4;
+  config.core.v3.HttpService http_service = 3;
```

Comment:
You can deprecate the previous `http_uri` and add new field `http_service`.

Reply:
not needed as discussed below